idd/Energy+.idd.in

```diff
@@ -38873,34 +38870,22 @@ AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan,
        \minimum 0.0
        \maximum 1.0
        \note fraction of cooling air flow rate
-  A3,  \field Air Inlet Node Name
```

Comment:
@mjwitte Reversed two of the four node names deprecated with the previous commit.  And updated idd, I/O Ref document, transition rule and example files.